spec/system/admin/stats_spec.rb

```diff
@@ -395,4 +395,24 @@
       end
     end
   end
+
+  context "SDG" do
+    scenario "Shows SDG stats link when SDG feature is enabled" do
+      visit admin_stats_path
+
+      expect(page).not_to have_link "SDG"
+
+      Setting["feature.sdg"] = "t"
```

Comment:
IMHO splitting it in two tests we don't lose any performance (there's no common setup for these tests) and we gain a bit of clarity:

```ruby
    scenario "Shows SDG stats link when SDG feature is enabled" do
      Setting["feature.sdg"] = true

      visit admin_stats_path

      expect(page).to have_link "SDG", href: sdg_admin_stats_path
    end

    scenario "Does not show SDG stats link when SDG feature is disbled" do
      Setting["feature.sdg"] = false

      visit admin_stats_path

      expect(page).not_to have_link "SDG"
    end
```

Also note we set the setting both times, so these tests do not depend on the default option (which we might change in the future).